experimental/traffic-portal/src/app/core/parameters/table/parameters-table.component.ts

```diff
@@ -0,0 +1,146 @@
+	templateUrl: "./parameters-table.component.html"
+})
+export class ParametersTableComponent implements OnInit {
+	/** List of parameters */
+	public parameters: Promise<Array<ResponseParameter>>;
+
+	/** Definitions of the table's columns according to the ag-grid API */
+	public columnDefs = [
+		{
+			field: "id",
+			headerName: "ID"
+		},
+		{
+			field: "configFile",
+			headerName: "Config File"
+		},
+		{
+			field: "name",
+			headerName: "Name"
+		},
+		{
+			field: "profiles",
+			headerName: "Profiles"
+		},
+		{
+			field: "secure",
+			headerName: "Secure"
+		},
+		{
+			field: "value",
+			headerName: "Value"
+		},
+		{
+			field: "lastUpdated",
+			headerName: "Last Updated"
+		}
+	];
+
+	/** Definitions for the context menu items (which act on augmented parameter data). */
+	public contextMenuItems: Array<ContextMenuItem<ResponseParameter>> = [
+		{
+			href: (responseParameter: ResponseParameter): string => `${responseParameter.id}`,
+			name: "Edit"
+		},
+		{
+			href: (responseParameter: ResponseParameter): string => `${responseParameter.id}`,
+			name: "Open in New Tab",
+			newTab: true
+		},
+		{
+			action: "delete",
+			multiRow: false,
+			name: "Delete"
+		},
+		{
+			href: (selectedRow: ResponseParameter): string => `/core/profiles/param/${selectedRow.id}`,
+			name: "Manage Profiles",
+			newTab: true,
+		}
+	];
+
+	/** A subject that child components can subscribe to for access to the fuzzy search query text */
+	public fuzzySubject: BehaviorSubject<string>;
+
+	/** Form controller for the user search input. */
+	public fuzzControl = new FormControl<string>("", {nonNullable: true});
+
+	constructor(private readonly route: ActivatedRoute, private readonly navSvc: NavigationService,
+		private readonly api: ParameterService, private readonly dialog: MatDialog, public readonly auth: CurrentUserService) {
+		this.fuzzySubject = new BehaviorSubject<string>("");
+		this.parameters = this.api.getParameters();
+		this.navSvc.headerTitle.next("Parameter");
+	}
+
+	/** Initializes table data, loading it from Traffic Ops. */
+	public ngOnInit(): void {
+		this.route.queryParamMap.subscribe(
+			m => {
+				const search = m.get("search");
+				if (search) {
+					this.fuzzControl.setValue(decodeURIComponent(search));
+					this.updateURL();
+				}
+			}
+		);
+	}
+
+	/** Update the URL's 'search' query parameter for the user's search input. */
+	public updateURL(): void {
+		this.fuzzySubject.next(this.fuzzControl.value);
+	}
+
+	/**
+	 * Handles a context menu event.
+	 *
+	 * @param evt The action selected from the context menu.
+	 */
+	public async handleContextMenu(evt: ContextMenuActionEvent<ResponseParameter>): Promise<void> {
+		const data = evt.data as ResponseParameter;
+		switch(evt.action) {
+			case "delete":
+				const ref = this.dialog.open(DecisionDialogComponent, {
+					data: {message: `Are you sure you want to delete parameter ${data.name} with id ${data.id} ?`, title: "Confirm Delete"}
```

Comment:
"Parameter" should be capitalized, "ID" is an initialism, so it should be in all caps. Also, extraneous space between the ID and the punctuation (<kbd>?</kbd>)